dev/docker/hive/Dockerfile

```diff
@@ -151,6 +151,10 @@ RUN chown -R datastrato:hadoop /home/datastrato
 # removed install packages
 RUN rm -rf /tmp/packages
 
+################################################################################
+# removed install cache
+RUN rm -rf /var/cache/apt/*
```

Comment:
hi @zhuzilong2013 Thank you for your contribution.

I think it's best to use the `apt clean` command to remove the cached files, but this needs to be tested.
https://askubuntu.com/questions/1350375/can-i-delete-var-cache-apt-archives

Reply:
Thanks for your comment. It's my mistake. I should use `rm -rf /var/lib/apt/list/*` actually.
In https://docs.docker.com/develop/develop-images/instructions/#run:
> Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.

I will fix and test it. I also want to move 'mysql-sever' to basic tools, what do you think?